CMakeLists.txt

```diff
@@ -7,6 +7,9 @@ if (MSVC AND MSVC_VERSION VERSION_LESS "1900")
     return()
 endif()
 
+#Set configurable install path
+set(CPACK_PACKAGING_INSTALL_PREFIX "/opt/rocm/hip" CACHE PATH "Package Installation path for HIP")
```

Comment:
Packaging is implemented only for Linuxes, whereas hipify-clang could be built for Windows as well. Thus, move this set down inside `if (UNIX)` condition, please.

Reply:
1. Currently, right. 2. In order not to change your line(s) when we start to support packaging for Windows. And why should the user see the Linux paths in its CMakeCache file on Windows? )

Reply:
Done